android/build.gradle

```diff
@@ -39,8 +41,8 @@ android {
     }
     defaultConfig {
         buildConfigField("boolean", "IS_NEW_ARCHITECTURE_ENABLED", isNewArchitectureEnabled().toString())
-        minSdkVersion safeExtGet("minSdkVersion", 21)
-        targetSdkVersion safeExtGet("targetSdkVersion", 33)
+        minSdkVersion safeExtGet("minSdkVersion", 23)
```

Comment:
Heads up, 0.76 will have a minSdkVersion of 24

Reply:
@alanjhughes This library is compatible with RN 0.73+, that's why I keep `minSdkVersion` to `23` for now.